blog/2024-07-10-uppy-4.0.md

```diff
@@ -407,14 +413,14 @@ For more information, see the [Companion docs](/docs/companion/).
 As a security measure, we now require the
 [`corsOrigins`](/docs/companion/#corsorigins) option to be set.
 
-It serves two purposes, it sets the `Access-Control-Allow-Origin` header and it
-sets the target origin for `window.postMessage()`, which is needed to
-communicate the OAuth token from the new tab you used to log-in to a provider
+It serves two purposes: it sets the `Access-Control-Allow-Origin` header as well
+as the target origin for `window.postMessage()`, which is needed to
+communicate the OAuth token from the new tab you used to log in to a provider
```

Comment:
The final bit of this sentence doesn't fully make sense to me: 'log in to a provider back to Companion'.

The content is going over my head a little bit, so can someone else think of a better way to write this (if that is even needed)?